package.json

```diff
@@ -13,35 +13,35 @@
     "prepare": "husky install"
   },
   "devDependencies": {
+    "@tailwindcss/postcss": "^4.1.5",
     "@tanstack/eslint-plugin-query": "^5.51.13",
     "@tanstack/react-query-devtools": "^5.59.20",
     "@types/node": "^20.16.1",
-    "@types/react": "^18.3.3",
+    "@types/react": "19.1.2",
     "@types/react-big-calendar": "^1.16.1",
-    "@types/react-dom": "^18.3.0",
+    "@types/react-dom": "19.1.3",
     "@types/uuid": "^10.0.0",
     "@typescript-eslint/eslint-plugin": "^8.8.0",
     "@typescript-eslint/parser": "^8.8.0",
-    "autoprefixer": "^10.4.16",
     "class-variance-authority": "^0.7.1",
     "clsx": "^2.1.1",
     "eslint": "^8.57.1",
```

Comment:
can we migrate this to be v9 and above, this does introduce some breaking changes we need to handle